edith-step/src/App.js

```diff
@@ -1,19 +1,46 @@
 import React, {Component} from 'react';
 import ReceiptInput from './ReceiptInput';
-// import "./App.css";
+import LineChart, {BarChart} from './UserChart';
+import axios from 'axios';
+//import "./App.css";
 
 
 /** Main webpage for the website. */
 class App extends Component {
-  /** Renders the main webpage. */
+  constructor() {
+    super();
```

Comment:
i think you need to pass props to these functions since this component has state

Reply:
Done.

Reply:
I think props and state are two different ways of managing variables. Since App isn't expecting information from props, I don't think it needs to be passed in.